executor/aggfuncs/aggfuncs.go

```diff
@@ -111,7 +111,7 @@ type AggFunc interface {
 	// return back. Aggregate operator implementation, no matter it's a hash
 	// or stream, should hold this allocated PartialResult for the further
 	// operations like: "ResetPartialResult", "UpdatePartialResult".
-	AllocPartialResult() PartialResult
+	AllocPartialResult() (PartialResult, int64)
```

Comment:
Add a comment for the `int64` return value, so as `Updatexx` and `Mergexx`

Reply:
done, PTAL